### DISCUSSION TITLE
Adding typelibs

### DISCUSSION THREAD

**Discussion by wqweto** [twinbasic/twinbasic#548] [General]
In PROJECT->Settings in `COM Type Library / ActiveX References` section the button `[Import from file...]` should probably add the newly imported typelib to `ENABLED COM REFERENCES` by default too, the way VBx does with the equivalent button.

Now it's mostly impossible to find the newly imported typelib in the long list of `ALL AVAILABLE COM REFERENCES` particularly if you don't actually know the name of the typelib the way they are listed there (you know only the `filename.tlb` you select in open file dialog).

**Comment by WaynePhillipsEA** [comment]
Moved to #553 thanks
